dataqs/forecastio/tasks.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import absolute_import
 
 from celery import shared_task
-from dataqs.forecastio.forecastio_air import ForecastIOAirTempProcessor
+from dataqs.aqicn.aqicn import AQICNProcessor
```

Comment:
@mbertrand same here. Did you change this by mistake or left over from initial debugging?